pages/pydata.md

```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyConUK and [PyData London](http://london.pydata.org/) are joining forces to deliver a data science focussed track at the conference. We will be hosting a number of talks and workshops about data science at the the conference, and we are looking for members of the PyData community to submit proposals to the [conference CFP](https://hq.pyconuk.org/cfp/proposals/new/).
+***
+
+## What we are looking for
+We are looking for speakers who wish to share their experiences of using Python in data science. Examples such as:
+
+  * How existing Python packages helped you to achieve your project goals:
+    * Data preparation
+    * Statistical analysis
+    * Machine learning
+    * Data visualisation
+  * Have you been working on developing your own Python applications for data science?
+    * Even if these are a _work-in-progress_, this is a great forum to present these!
+    
+The time allocated for each talk is 25 minutes. If you are happy to receive any questions from the audience (this is not compulsory), then please allow time for these.
+    
+## Previous PyData talks
+If anyone would like to see examples of the sorts of talks that have been presented at PyData conferences in UK the in the past, these can be seen on the [PyData YouTube channel](https://www.youtube.com/user/PyDataTV).
+    
+## Community Spirit
+Both PyConUK and PyData are thriving communities that want to help people to learn and develop their tech skills. We welcome talks from all backgrounds and all abilities, across both academia and industry. Even if this your first talk at a tech conference, you will be given a warm welcome, with many people happy to provide any assistance or guidance that you require.
```

Comment:
Up to you, but I wonder if it might be nice to have a link to http://2017.pyconuk.org/diversity-accessibility-inclusion/ in this paragraph?